### PR TITLE
Update docker-basics-dotnet-core.md

### PR DESCRIPTION
**Pull request by jahbenjah**
## Summary
Update docker installation links
Updtate from dotnet 2.0 to 2.1
Fixes #Issue_Number (if available)

### CODE REVIEW

**Review by kendrahavens — APPROVED** [review]
LGTM! Thank you!

 A small nit, referring to Docker "CE" might confuse users who aren't familiar with the Docker client offerings, but it's not a big issue.

**Review by kendrahavens — APPROVED** [review]
Thanks!

File: docs/core/docker/docker-basics-dotnet-core.md [hunk 1]
```diff
@@ -33,11 +33,11 @@ You can build both Windows and Linux containers with [multi-arch based tags](htt
 
 To complete this tutorial:
 
-#### .NET Core 2.0 SDK
+#### .NET Core 2.1 SDK
 
-* Install [.NET Core SDK 2.0](https://www.microsoft.com/net/core).
+* Install [.NET Core SDK 2.1](https://www.microsoft.com/net/core).
```
Comment (mairaw) [inline docs/core/docker/docker-basics-dotnet-core.md]: wondering if sending them straight to downloads wouldn't be better:
https://www.microsoft.com/net/download

File: docs/core/docker/docker-basics-dotnet-core.md [hunk 2]
```diff
@@ -33,11 +33,11 @@ You can build both Windows and Linux containers with [multi-arch based tags](htt
```
Comment (mairaw) [inline docs/core/docker/docker-basics-dotnet-core.md]: Can you also remove lines 4 and 5 (author and ms.author metadata) and update ms.date to 10/11/2018?

**Review by mairaw — COMMENTED** [review]
This looks great @jahbenjah. I left a couple of comments for you to consider.

File: docs/core/docker/docker-basics-dotnet-core.md (hunk 2 above)
Comment (jahbenjah) [reply docs/core/docker/docker-basics-dotnet-core.md]: Done

**Comment by Thraka** [comment]
@mairaw I think this is good to go. I don't want to just approve it since you did have feedback. I do think he has addressed your feedback though.

**Review by mairaw — APPROVED** [review]
Yes @Thraka! Thanks for bringing this my attention. Thank you @jahbenjah for your contribution.

**Pull request merged**